src/components/AttachmentViewer/AttachmentViewer.tsx

```diff
@@ -32,7 +31,9 @@ import {
 export interface Attachment {
```

Comment:
There already is an Attachment type. Extract that instead

Reply:
As discussed, made a story to use the correct Attachement type in INcidentdetail of INcident-management, incidentdetail of my-incidents and AttachmentViewer. https://gemeente-amsterdam.atlassian.net/browse/SIG-5454